core/consensus/babe/impl/block_executor_impl.cpp

```diff
@@ -182,12 +194,16 @@ namespace kagome::consensus {
     const auto &previous_best_block = previous_best_block_res.value();
 
     OUTCOME_TRY(core_->execute_block(block_without_seal_digest));
+
     auto exec_end = std::chrono::high_resolution_clock::now();
-    SL_DEBUG(logger_,
-             "Core_execute_block: {} ms",
-             std::chrono::duration_cast<std::chrono::milliseconds>(exec_end
-                                                                   - exec_start)
-                 .count());
+    auto taken_milliseconds =
```

Comment:
No need to fix this, but in my view this is duration. (duration/operation duration/execution duration...  "whatever" duration)
```suggestion
    auto duration_ms =
```